src/services/websockets.js

```diff
@@ -20,26 +20,32 @@ export function initialise (actionCreatorBinder) {
   } = actionCreatorBinder({...SearchResultActions, ...TagActions});
   primus.on('data', function received (data) {
     console.log('incoming socket data', data);
-    if (data.connection) {
-      saveSocketConnectionId(data.connection);
+    if (data.graphql) {
+      saveSearchResult(data);
+    }
+  });
+
+  primus.once('open', (data) => {
```

Comment:
@lennym yes, and on reflection, this does not *look* like it will affect `Tracy` because you are still `join`ing the room just making sure it's only done `once`. 👍